LICENSE

```diff
@@ -17,3 +17,7 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+Contributors:
+
+Guillermo Alvarado - guillermo@sentinel.la
```

Comment:
please do not change this file. The comment in CONTRIBUTING.md refers to adding your name to copyright *header* of the source files. There's no such header in the README file that you are changing, so other changes are necessary.

Reply:
Ok @yurishkuro, undoing changes on this file..